test/e2e/framework.go

```diff
@@ -172,7 +172,7 @@ func (f *Framework) WriteFileViaContainer(podName, containerName string, path st
 		}
 	}
 	command := fmt.Sprintf("echo '%s' > '%s'", contents, path)
-	stdout, stderr, err := kubectlExec(f.Namespace.Name, podName, containerName, "--", "/bin/sh", "-c", command)
+	stdout, stderr, err := f.kubectlExec(f.Namespace.Name, podName, containerName, "--", "/bin/sh", "-c", command)
```

Comment:
/cc @quinton-hoole on these changes, since they seem unnecessary and he's been working on the framework code lately.